test/tictactoe/Bot_test.exs

```diff
@@ -0,0 +1,78 @@
+defmodule BotTests do
+  use ExUnit.Case
+
+  describe "calculate move" do
+    test "player already won - do nothing" do
+      board = Fixtures.randomBoard
+      first_player = Forge.player
+      second_player = Forge.player
+
+      assert Bot.calculate_move(:PlayerWon, board, first_player, second_player, :ANY_TYPE) == nil
+    end
+
+    test "there is already a draw - do nothing" do
```

Comment:
Could flip it to make the it a simple sentence 
test “does nothing if there is already draw present”